### PR TITLE
[HIRFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by MahMaejah**

### CODE REVIEW

File: submissions/chipoHameja/database/transportation_db.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	id INT NOT NULL AUTO_INCREMENT,
+    passenger_id INT NOT NULL,
+    passenger_class ENUM('1', '2', '3') NOT NULL,
+    passenger_ticket_number VARCHAR(250) NOT NULL,
+    trip_fare FLOAT(10, 2) NOT NULL,
+    assigned_cabin VARCHAR(250),
+    number_of_parents_children_aboard INT NOT NULL,
+    number_of_siblings_spouses_aboard INT NOT NULL,
+    point_of_embarkation VARCHAR(250) NOT NULL,
+    accident_occured BOOLEAN NOT NULL,
```
Comment (ToluEnny24) [inline submissions/chipoHameja/database/transportation_db.sql:25]: accident_occured is not necessary. It is the survival status that is required.,

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Remove the accident_occured

File: submissions/chipoHameja/database/transportation_db.sql (hunk 1 above)
Comment (MahMaejah) [reply submissions/chipoHameja/database/transportation_db.sql]: @ToluEnny24 I have made requested changes

**Review by ToluEnny24 — APPROVED** [review]
Changes effected. Well done...

**Pull request merged**